src/scripts/activity.js

```diff
@@ -5,24 +5,24 @@ class Activity {
         if (this.isValidPage(tab) === true) {
             if (tab.id && (tab.id != 0)) {
                 tabs = tabs || [];
-                var url = new Url(tab.url);
-                var isDifferentUrl = false;
+                let url = new Url(tab.url);
```

Comment:
Nice to see here Url but it is supported by a few last versions of browsers. Let's leave it as it is for now, and if users complain, we will use webpack with babel.